test/common/test_status_code.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright $year Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from zepben.auth.common.status_code import StatusCode
+
+
+def test_status_code():
+    assert len(StatusCode) == 5
+
+    assert StatusCode.OK.value == 200
+    assert StatusCode.MALFORMED_TOKEN.value == 400
+    assert StatusCode.UNAUTHENTICATED.value == 403
+    assert StatusCode.PERMISSION_DENIED is StatusCode.UNAUTHENTICATED
```

Comment:
Just to confirm, do we want these to be aliases? They won't be distinguishable if so.

Reply:
(They are both set to 403 in the JVM version too)

Reply:
Is one of them meant to be 401? PERMISSION_DENIED feels like it should be 401?

What's the purpose of using our own StatusCode? Why did you choose that over just using grpc.StatusCode?

Reply:
Honestly I was just blindly transliterating. I'll use grpc.StatusCode instead.

Reply:
Class removed.